backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersController.java

```diff
@@ -50,4 +54,27 @@ public UserModel getUserById(
 		return userModelAssembler.toModel(user);
 	}
 
+
+    @PostMapping({ "/{userId}/email-validations"})
+    @Operation(summary = "Verify the status of a confirmation code.", operationId="verify-confirmation-code")
+    public ResponseEntity<?>  verifyConfirmationCodeStatus(
```

Comment:
This can return `void` with a `@ResponseStatus` annotation.

Reply:
Implemented in commit 2cebd2d6f0685660fa512c66c4a5da16869070e1.